app/models/smart_annotation.rb

```diff
@@ -0,0 +1,117 @@
+class SmartAnnotation
```

Comment:
Does this make a sense to be under `models`, since it's not mapped to a DB table?